storybook-example/stories/SecureBiometricInterfaceIntegrator.stories.tsx

```diff
@@ -57,17 +58,17 @@ const sbiEnv = {
 const sbiEnvType =
   '{\r\n  env: "Staging" | "Developer" | "Pre-Production" | "Production",\r\n  captureTimeout: "number",\r\n  irisBioSubtypes: "Left" | "Right" | "UNKNOWN",\r\n  fingerBioSubtypes:\r\n    "Left IndexFinger" | "Left MiddleFinger" | "Left RingFinger" | "Left LittleFinger" | "Left Thumb" | "Right IndexFinger" | "Right MiddleFinger" | "Right RingFinger" | "Right LittleFinger" | "Right Thumb" | "UNKNOWN",\r\n  faceCaptureCount: 1,\r\n  faceCaptureScore: "0-100",\r\n  fingerCaptureCount: "1-10",\r\n  fingerCaptureScore: "0-100",\r\n  irisCaptureCount: "1-2",\r\n  irisCaptureScore: "0-100",\r\n  portRange: "4501-4600",\r\n  discTimeout: "number",\r\n  dinfoTimeout: "number",\r\n  domainUri: "string",\r\n}';
 
-export default {
-  title: "Secure Biometric Interface",
+const SBIMeta = {
+  title: "JavaScript/Secure Biometric Interface",
   tags: ["autodocs"],
-  render: (args) => renderComponent(args),
+  component: SecureBiomtericInterfaceIntegrator,
   argTypes: {
-    container: {
-      control: "object",
-      type: { required: true, name: "HTMLElement" },
-      description:
-        "Container inside of which the whole component will be created",
-    },
+    //   container: {
```

Comment:
if not required, can we remove this commented parameter

Reply:
resolved in this [ca7c463](https://github.com/mosip/mosip-plugins/pull/56/commits/ca7c463b799a7e2584b2318c9bd15a2fa66880e7)